.github/ISSUE_TEMPLATE/ga_request.md

```diff
@@ -11,22 +11,22 @@ assignees: ''
 
 Name and link to repository
 
-### Does the repository follow the latest tagged minor release in [GDI specification](https://github.com/signalfx/gdi-specification/blob/v1.4.0/specification/repository.md)?
+### Does the repository follow the latest tagged minor release in [GDI specification](https://github.com/signalfx/gdi-specification/blob/v1.5.0/specification/repository.md)?
 
-- [ ] Has an appropriate [maintainers team](https://github.com/signalfx/gdi-specification/blob/v1.4.0/specification/repository.md#teams).
-- [ ] [Permissions](https://github.com/signalfx/gdi-specification/blob/v1.4.0/specification/repository.md#permissions)
+- [ ] Has an appropriate [maintainers team](https://github.com/signalfx/gdi-specification/blob/v1.5.0/specification/repository.md#teams).
+- [ ] [Permissions](https://github.com/signalfx/gdi-specification/blob/v1.5.0/specification/repository.md#permissions)
```

Comment:
This kind of links will no go through lychee validation.
Consider to make it as follow up pr after official release.

Reply:
https://github.com/signalfx/gdi-specification/pull/226/commits/9dbffff90dae7ad8e325a20cdbde58fc56ed21d8 Done here as I do not want to use "admin powers"